content/en/runtime.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A runtime, in general, is something that executes a piece of software.
+It is an abstraction of the underlying operating system that knows how to translate the program's commands into respective actions on the operating system level. 
```

Comment:
```suggestion
It is an [abstraction](/abstraction/) of the underlying operating system that translates the program's commands into respective actions on the operating system. 
```
Do we need "level?" It confused me a bit. As far as I understood, an abstraction layer will execute things for the app without the app needing to know what's happening behind the "abstraction curtain." So it's simply executing on the app's behalf. 

Also, does it go both ways or just from the app to the OS?

Reply:
```suggestion
It is an [abstraction](/abstraction/) of the underlying operating system that translates the program's commands into respective actions for the operating system. 
```

If we're taking out the reference to level, then it wouldn't be "on the" anymore. Good call linking out to abstraction.

> Also, does it go both ways or just from the app to the OS?

I think info would go both ways, but I'm not sure we need to explain that here.